### DISCUSSION TITLE
Cardinal Modules in VCV

### DISCUSSION THREAD

**Discussion by moon47usaco** [DISTRHO/Cardinal#422] [Q&A] [answered]
Can modules like Sassy Scope and glBars be built to work with VCV.

If possible how much work needs to be done. Could the modules code just be separated and built to work with VCV or would the code need a massive rewrite to work. I would love to have these and the text editor in VCV.

**Comment by dromer** [comment, marked as answer]
They could, but VCV web store does not allow free software binaries to be sold by developers so there isn't any incentive to make this happen currently.

Of course the code is still FOSS so you could fork it and make them yourself.
